pom.xml

```diff
@@ -325,6 +325,22 @@
             <trimStackTrace>false</trimStackTrace>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-plugin-plugin</artifactId>
+          <version>${project.version}</version>
+          <configuration>
+            <internalJavadocBaseUrl>./apidocs/</internalJavadocBaseUrl>
+            <externalJavadocBaseUrls>
+              <externalJavadocBaseUrl>https://docs.oracle.com/javase/8/docs/api/</externalJavadocBaseUrl>
+            </externalJavadocBaseUrls>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-plugin-report-plugin</artifactId>
+          <version>${project.version}</version>
+        </plugin>
```

Comment:
IMHO we can't use it now in this way ... there is egg and hen problem
We should use the latest released version

The `maven-plugin-report-plugin` has not been released yet